flashinfer/gemm/gemm_base.py

```
@@ -5935,14 +5946,22 @@ def forward(
+            # NOTE: upstream b12x ctor inserts mma_k/tile_k/single_work_tile_per_cta
+            # before use_prefetch, so pass these by keyword to avoid mis-binding.
             make_kernel = lambda: Sm120B12xBlockScaledDenseGemmKernel(
                 sf_vec_size,
                 mma_tiler_mn,
                 cluster_shape_mn,
-                use_prefetch,
-                enable_pdl,
+                use_prefetch=use_prefetch,
+                enable_pdl=enable_pdl,
+                swap_ab=swap_ab,
             )
```

Ruff (0.15.15)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `make_kernel` as a `def`

(E731)